lib/gauge_runtime.rb

```diff
@@ -72,15 +72,16 @@ def self.port_from_env_variable(env_variable)
     end
 
     STDOUT.sync = true
-    GaugeLog.init
     StaticLoader.load_files(DEFAULT_IMPLEMENTATIONS_DIR_PATH)
     if ENV.key? 'GAUGE_LSP_GRPC'
+      GaugeLog.debug 'Starting grpc server..'
       server = GRPC::RpcServer.new
       p = server.add_http2_port('127.0.0.1:0', :this_port_is_insecure)
       server.handle(Gauge::LSPServer.new(server))
-      puts "Listening on port:#{p}"
+      GaugeLog.info "Listening on port #{p}"
```

Comment:
Why the `:` has been removed? This line is used by `vs code` (LSP) to extract the port where the gRPC server is running. Removing `:` might cause the issue and the features might not work.

Reply:
It was removed mistakenly. Will make the changes